common-libs/internal-api-clients/src/openapi/_affinidiVault.ts

```diff
@@ -0,0 +1,297 @@
+/* eslint-disable */
```

Comment:
You mean generated with an npm script?

Right now everything between `export default` and `as const` is just a copy of https://affinidi-vault.dev.affinity-project.org/api/swagger as-is

Reply:
yes, so in case something changed in definitions for some service would be nice to keep them in sync at sdk. Just regenerate (get new copies) during build.

Reply:
Maybe later, in another task.